### PR TITLE
Convert Message-Star Package to js 

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-message-star/client/views/starredMessages.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/*globals StarredMessage */
```
Comment (rodrigok) [inline packages/rocketchat-message-star/client/views/starredMessages.js]: I can't find the coffee version of this file

File: packages/rocketchat-message-star/server/publications/starredMessages.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Meteor.publish('starredMessages', function(rid, limit) {
+	if (limit == null) {
```
Comment (rodrigok) [inline packages/rocketchat-message-star/server/publications/starredMessages.js]: Use default values here

File: packages/rocketchat-message-star/client/views/starredMessages.js (hunk 1 above)
Comment (MartinSchoeler) [inline packages/rocketchat-message-star/client/views/starredMessages.js]: https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-message-star/client/views/starredMessages.coffee

Comment (rodrigok) [inline packages/rocketchat-message-star/client/views/starredMessages.js]: Can you delete the file in your commit? Or do you want to keep the old .coffee file?

File: packages/rocketchat-message-star/client/views/starredMessages.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	hasMore() {
+		return Template.instance().hasMore.get();
+	}
+});
+
+Template.starredMessages.onCreated(function() {
+	this.hasMore = new ReactiveVar(true);
+	this.limit = new ReactiveVar(50);
+	return this.autorun(() => {
+		return () => {
```
Comment (rodrigok) [inline packages/rocketchat-message-star/client/views/starredMessages.js]: Why this function? It'll never be called

File: packages/rocketchat-message-star/client/views/starredMessages.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+			const sub = this.subscribe('starredMessages', this.data.rid, this.limit.get());
+			if (sub.ready()) {
+				if (StarredMessage.find({
```
Comment (rodrigok) [inline packages/rocketchat-message-star/client/views/starredMessages.js]: Do not break the if, move the find to a variable

**Pull request merged**